README.md

```diff
@@ -25,7 +25,7 @@ Note that packet dissection and creation was split from SharpPcap some years ago
   * Live device lists
   * Statistics
   * Reading packets from Live Devices (actual network devices) and Offline Devices (Capture files)
-  * Support for Berkley Packet Filters
+  * Support for [Berkley Packet Filters](https://www.tcpdump.org/manpages/pcap-filter.7.html)
```

Comment:
Please keep in mind that WinDivert uses a different syntax for filtering

Reply:
Should we note that?

Reply:
In `WinDivertDevice.Filter` at least.